web/client/utils/CoordinatesUtils.js

```diff
@@ -56,6 +57,100 @@ let crsLabels = {
     "EPSG:3857": "EPSG:3857"
 };
 
+const normalizePoint = (point) => {
```

Comment:
You moved this, did you change anything?

Reply:
no, i didn't change the function